index.js

```diff
@@ -6,7 +6,7 @@ const LineDiff = require('line-diff');
 const bannerMessage =
   '// This file is automatically generated.\n// Please do not change this file!';
 
-const cssModuleExport = 'declare const cssExports: CssExports;\nexport = cssExports;\n';
+const cssModuleExport = 'export const cssExports: CssExports;\nexport default cssExports;\n';
```

Comment:
Why r u exporting `const cssExports`? There is no such actually.

May be more accurate will be smthing like this:
```js
const cssModuleExport = 'const cssExports: CssExports;\nexport default cssExports;\n';
```
?